cnjfacil/testes/extrator.py

```diff
@@ -56,10 +56,23 @@ def test_cnj_incorrigivel(self):
         extrator = ExtratorCNJ(texto, maximo_tentativas=0)
         self.assertEqual(extrator.cnjs, list())
 
-    def test_cnjs_repetidos_aprecem_apenas_uma_vez(self):
+    def test_cnjs_repetidos_aparecem_apenas_uma_vez(self):
         texto = '''0053087-35.2013.8.13.0693 texto
                 0516710-11.2017.8.13.0000 e 0516710-11.2017.8.13.0000
                 '''
         extrator = ExtratorCNJ(texto)
         self.assertEqual(extrator.cnjs, ['0053087-35.2013.8.13.0693',
                                          '0516710-11.2017.8.13.0000'])
+
+    def test_nao_adiciona_valores_com_ano_do_cnj_invalido(self):
```

Comment:
Seria também interessante reduzir a sample desse test. Não precisamos de tanto texto aqui.

Deixe apenas os CNJs separados por uma pequena quantidade de testes para validar sua implementação.

Isso facilita para quem for entender seu teste no futuro.

Reply:
Para o caso proposto no teste, o tamanho do texto realmente não faz diferença.
O texto foi reduzido.
Obrigado.